examples/todo/data/database.js

```diff
@@ -30,41 +30,41 @@ const todoIdsByUser = {
 };
 
 const notifiers = [];
-let nextTodoId = 0;
-addTodo('Taste JavaScript', true);
-addTodo('Buy a unicorn', false);
 
 function notifyChange(topic, data) {
-  notifiers.forEach(notifier => notifier({topic, data}));
+  // Delay the change notification to avoid the subscription update hitting the
+  // client before the mutation response.
+  setTimeout(() => {
+    notifiers.forEach(notifier => notifier({ topic, data }));
+  }, 100);
```

Comment:
The flash of two TODOs was because the subscription response came back before the mutation response. In a "real" system this is probably unlikely. This hard-coded delay simulates that, though.